api/server/structs/endpoints_validator.go

```diff
@@ -18,6 +18,10 @@ type SubmitAggregateAndProofsRequest struct {
 	Data []*SignedAggregateAttestationAndProof `json:"data"`
 }
 
+type SubmitAggregateAndProofsRequestV2 struct {
```

Comment:
is this type needed? 🤔

Reply:
I think we can just modify the existing request type and set its `Data` field to be of type `[]json.RawMessage`

Reply:
I think we can just modify the existing request type and set its `Data` field to be of type `[]json.RawMessage`